app/models/concerns/smooch_strings.rb

```diff
@@ -1,336 +1,15 @@
-          "id": "Anda saat ini sedang berlangganan ke buletin kami.",
-          "be": "আপনি বর্তমানে আমাদের নিউজলেটারে সাবস্ক্রাইব করেছেন।",
-          "fr": "Vous êtes actuellement abonné à notre lettre d’information.",
-          "de": "Sie haben unseren Newsletter derzeit abonniert.",
-          "hi": "आपने इस समय हमारे न्यूज़लेटर को सब्स्क्राइब किया हुआ है।",
-          "kn": "ನೀವು ಈಗ ನಮ್ಮ ನ್ಯೂಸ್ ಲೆಟರ್‌ಗೆ ಸಬ್‌ಸ್ಕ್ರೈಬ್ ಆಗಿರುತ್ತೀರಿ",
-          "mr": "आपण सध्या आमचे वृत्तपत्र सबस्क्राइब केले आहे.",
-          "pt": "Atualmente, você estás inscrito(a) em nosso boletim informativo.",
-          "pa": "ਤੁਸੀਂ ਇਸ ਸਮੇਂ ਸਾਡੇ ਨਿਊਜ਼ਲੈਟਰ ਨੂੰ ਸਬਸਕ੍ਰਾਈਬ ਕੀਤਾ ਹੋਇਆ ਹੈ",
-          "es": "Actualmente estás suscrita(o) a nuestro boletín.",
-          "ta": "தற்சமயம் நீங்கள் எங்கள் செய்தி மடலைப் பெறுவதற்குப் பதிவு செய்திருக்கிறீர்கள்",
-          "te": "మీరు ఇప్పుడు మా వార్తాలేఖకు సభ్యత్వాన్ని పొంది ఉన్నారు.",
-          "ur": "آپ فی الحال ہمارے نیوز لیٹر کو سبسکرائب کر چکے ہیں۔"
-        },
-        "unsubscribe_button_label": {
-          "en": "Unsubscribe",
-          "id": "Brhenti brlangganan",
-          "be": "আনসাবস্ক্রাইব করুন",
-          "fr": "Me désabonner",
-          "de": "Abbestellen",
-          "hi": "अनसब्स्क्राइब करें",
-          "kn": "ಅನ್ಸಬ್‌ಸ್ಕ್ರೈಬ್",
-          "mr": "अनसबस्क्राइब करा",
-          "pt": "Desinscrever-se",
-          "pa": "ਅਨਸਬਸਕ੍ਰਾਈਬ ਕਰੋ",
-          "es": "Cancelar suscripción",
-          "ta": "பதிவை ரத்து செய்",
-          "te": "అన్‌సబ్‌స్క్రైబ్",
-          "ur": "رکنیت ختم کریں۔"
-        },
-        "unsubscribed": {
-          "en": "You are currently not subscribed to our newsletter.",
-          "id": "Saat ini Anda tidak berlangganan buletin kami.",
-          "be": "আপনি বর্তমানে আমাদের নিউজলেটার সদস্যতা নেই.",
-          "fr": "Vous n’êtes actuellement pas abonné à notre lettre d’information.",
-          "de": "Sie haben unseren Newsletter derzeit nicht abonniert.",
-          "hi": "आपने वर्तमान में हमारे न्यूज़लेटर की सदस्यता नहीं ली है।",
-          "kn": "ನೀವು ಪ್ರಸ್ತುತ ನಮ್ಮ ಸುದ್ದಿಪತ್ರಕ್ಕೆ ಚಂದಾದಾರರಾಗಿಲ್ಲ.",
-          "mr": "तुम्ही सध्या आमच्या वृत्तपत्राचे सदस्यत्व घेतलेले नाही.",
-          "pt": "Atualmente, você não está inscrito(a) em nosso boletim informativo.",
-          "pa": "ਤੁਸੀਂ ਵਰਤਮਾਨ ਵਿੱਚ ਸਾਡੇ ਨਿਊਜ਼ਲੈਟਰ ਦੀ ਗਾਹਕੀ ਨਹੀਂ ਲਈ ਹੈ।",
-          "es": "Actualmente no estás suscrita(o) a nuestro boletín.",
-          "ta": "தற்சமயம் நீங்கள் எங்கள் செய்தி மடலைப் பெறும் வசதியை ரத்து செய்து இருக்கிறீர்கள்",
-          "te": "మీరు ప్రస్తుతం మా వార్తాలేఖకు సభ్యత్వం పొందలేదు.",
-          "ur": "آپ نے فی الحال ہمارے نیوز لیٹر کو سبسکرائب نہیں کیا ہے۔"
-        }
-      }[key.to_sym]
-      language = language.gsub(/[-_].*$/, '').to_sym
-      string ? (string[language] || string[:en]) : string
+      yaml = YAML.load(File.read(File.join(Rails.root, 'config', 'tipline_strings.yml')))
+      language = language.gsub(/[-_].*$/, '')
+      strings = yaml[language] || yaml['en']
+      strings[key]
```

Comment:
As per the previous code, please fallback to `key` when there is no `strings[key]`: `strings[key] || key`.

Also, please confirm if `YAML.load` loads the keys as strings (e.g., `'en'`) or symbol (e.g., `:en`).